frontend/src/views/CellGuide/components/TissueCard/index.tsx

```diff
@@ -141,7 +159,7 @@ export default function TissueCard({ description, name }: Props): JSX.Element {
         {/* This prevents auto zooming on the input box on mobile */}
         <meta
           name="viewport"
-          content="width=device-width, initial-scale=1, minimum-scale=1, maximum-scale=1"
```

Comment:
for my own education, why is `maximum-scale=1` removed?

Reply:
I got a linting error:
```
axe-linter (meta-viewport): Ensures <meta name="viewport"> does not disable text scaling and zoomingaxe-linter[meta-viewport](https://dequeuniversity.com/rules/axe/4.8/meta-viewport?application%3Daxe-linter)
```

Reply:
I didn't notice any difference after removing it in mobile view...Not sure what that meta tag is even doing.